--- ../orig/tomatoes-1.55/src/soundmusic.cpp	2004-11-21 11:18:29.000000000 +0100
+++ ./src/soundmusic.cpp	2009-01-03 00:51:32.000000000 +0100
@@ -205,7 +205,11 @@
 	
 	// Initialize the Mixer	
 	if(Mix_OpenAudio(config.sound_freq, AUDIO_S16, 2, 4096))	// Frequency, 16 bit sound, channels (stereo), buffer size
-		error_msg("Unable to open the audio device!\nError: %s\n", Mix_GetError());
+	{
+		fprintf( stderr, "Unable to open the audio device. Turning off sound.\n" );
+		config.sound = 0;
+		return;
+	}
 	
 	// Load the sounds
 	load_sound("snd_appear.wav");
